Makefile

```diff
@@ -1,10 +1,10 @@
-world: compile
+world: node_modules
 	node_modules/.bin/vsce package
 
 clean:
-	rm -rf node_modules out
+	rm -rf node_modules client/out server/out html_views/out
```

Comment:
This line can be replaced by a terse `git clean -dfx`, if the `.gitignore` is correct.